src/containers/LessonsContainer/LessonsContainer.js

```diff
@@ -122,8 +130,18 @@ const LessonsContainer = () => {
             )}
           />
         </Grid>
+        <Grid item>
+          <Button
+            variant='contained'
+            onClick={(e) => openPaymentForm(e)}
+            disabled={lesson.groupId  === '' || lesson.groupId === undefined}
```

Comment:
Вроде оставлял уже коммент, там не заметила наверно здесь преобразование в bool с инверсией
```suggestion
            disabled={!lesson.groupId}
```